### ISSUE TITLE
程序无法运行，bundling failed: Error: Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by 749801324** [zhanguangao/react-native-ble-plx-demo#3]
若降低react-native版本，则会报出bundling failed: Error: Plugin 0 specified in "D:\\react-native-ble-plx-demo-master\\node_modules\\babel-preset-react-native\\index.js" provided an invalid property of "default"。

**Comment by 749801324** [comment]
下载代码，
yarn
react-native link
react-native run-android
app创建成功，但运行报错。
使用测试机器为魅族手机

**Comment by zhanguangao** [comment]
react native 是0.59.9版本吗？

**Comment by 749801324** [comment]
可能会更低，我尝试了0.57.2 和0.57.8 还有0.59.9

**Comment by zhanguangao** [comment]
你尝试创建一个空项目，然后再集成react-native-ble-plx

**Comment by 749801324** [comment]
但是你的项目，拉取下来后会报错。我正在尝试重新创建。

**Comment by HeiSiBaby** [comment]
请问解决了么？我现在也有同样的问题

**Comment by zhanguangao** [comment]
@HeiSiBaby demo的RN版本有点旧了，可以先创建空项目再集成

**Comment by HeiSiBaby** [comment]
> @HeiSiBaby demo的RN版本有点旧了，可以先创建空项目再集成
我重新集成了 但是在连接的时候报了一个原生交互的错误 大概意思是map不能转换成string

**Comment by zhanguangao** [comment]
已将demo的react-native升级到0.61.2， react-native-ble-plx升级到1.1.0，请使用Xcode11，Android Studio 3.2及以上版本编译
